aclk/aclk.c

```diff
@@ -243,7 +245,7 @@ static void msg_callback(const char *topic, const void *msg, size_t msglen, int
 static void puback_callback(uint16_t packet_id)
 {
     if (++aclk_pubacks_per_conn == ACLK_PUBACKS_CONN_STABLE)
-        aclk_reconnect_delay(0, 0, 0, 0);
+        aclk_tbeb_delay(0, 0, 0, 0);
```

Comment:
We can (1) add a macro with the appropriate name that denotes the intent, (2) add inline comment strings for each arg, (3) add a single comment explaining what we want to achieve with this call. Same for the other calls using constants.

Reply:
added macro for readability